packages/embed/e2e/cypress-utils.ts

```diff
@@ -3,7 +3,7 @@ export type Viewport = {
   height: number
 }
 
-export const IFRAME_SELECTOR = '[data-qa="iframe"]'
+export const IFRAME_SELECTOR = 'iframe'
```

Comment:
I dont really see any added value from this const 😃

Reply:
Yeah me neither, I think it was there already. I saw the utils file and added it as it might have been used in other tests. I i will remove it